lib/order.rb

```diff
@@ -0,0 +1,72 @@
+#require_relative '..lib/customer'
+class Order
+  @@orders = []
+
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status=:pending)
+    @id = id
+    # if @id.class != integer
+    #   puts "not a num"
+    # end
+
+    @products = products
+    # if @product_cost.class != hash
+    #   puts "not a hash"
+    # end
+
+    @customer = customer
+
+    @fulfillment_status = fulfillment_status
+
+    raise ArgumentError unless
+    @fulfillment_status == :pending || @fulfillment_status == :paid || @fulfillment_status == :processing || @fulfillment_status == :shipped || @fulfillment_status == :complete
```

Comment:
This is a cool syntax you did, using the `unless`! I like it. There might be a better way to shorten if `@fulfillment_status` is one of many different options somehow...